examples/source/advertising-analytics/Tracking_Core_Web_Vitals.html

```diff
@@ -99,3 +99,48 @@
       }
     </script>
   </amp-analytics>
+
+
+  <!--
+  ## Comparing performance on Cache vs Origin
+  -->
+  <!--
+    AMP Caches can improve your code with a number of optimizations, but it is important to not rely on a Cache for your site's performance. A good way to make sure you are delivering a great experience for all your users is to keep track of how your pages' Core Web Vitals are performing on and off of the Cache.
+
+We can build on our Google Analytics to do this. One of substitutions provided by amp-analytics is `ampdocHost` - this lets us track the URL of the page the user visited. We exand our earlier config using `extraUrlParams`. From there, we need to [set up](https://support.google.com/analytics/answer/2709829) a new Custom Dimension to track this. Make sure to take note of the number of the Custom Dimension - below we assume it was "1", in `cd1`..
```

Comment:
who's `our` referring to?

Reply:
`this lets us track the URL` doesn't this only track the domain?

Reply:
"Make sure to take note of the number of the Custom Dimension" where is this number coming from?

Reply:
> who's our referring to?

the "reader and author", I suppose. I was trying to say we will expand on the example just discussed in the section right above this one. updated

> this lets us track the URL 

Yes, phrasing as poor.  updated.

> "Make sure to take note of the number of the Custom Dimension" where is this number coming from?

google analytics generates it. I will update with screenshot you request to make this more clear.

Reply:
thanks!